trieste/acquisition/rule.py

```diff
@@ -1068,3 +1081,118 @@ def state_func(
             return state_, points
 
         return state_func
+
+
+class BatchHypervolumeSharpeRatioIndicator(
+    AcquisitionRule[TensorType, SearchSpaceType, ProbabilisticModelType]
+):
+    def __init__(
+        self, batch_size: int = 5, population_size: int = 500, filter_threshold: float = 0.4
+    ):
+        if pymoo is None:
+            raise ImportError(
+                "BatchHypervolumeSharpeRatioIndicator requires pymoo, "
+                "which can be installed via `pip install trieste[qhsri]`"
+            )
+        builder = ProbabilityOfImprovement().using(OBJECTIVE)
+
+        self._builder: AcquisitionFunctionBuilder[ProbabilisticModelType] = builder
+        self._batch_size: int = batch_size
+        self._population_size: int = population_size
+        self._filter_threshold: float = filter_threshold
+        self._acquisition_function: Optional[AcquisitionFunction] = None
+
+    def __repr__(self) -> str:
+        """"""
+        return f"""BatchHypervolumeSharpeRatioIndicator(
```

Comment:
Why not include the other parameters too?

```suggestion
        return f"""BatchHypervolumeSharpeRatioIndicator(batch_size={batch_size}, population_size={population_size}, filter_threshold={filter_threshold})"""
```